src/main/java/org/opentripplanner/transit/model/site/StopLocation.java

```diff
@@ -43,6 +44,9 @@ public interface StopLocation extends LogInfo {
   @Nullable
   I18NString getUrl();
 
+  @Nonnull
+  String getStopType();
```

Comment:
This is specific to the Transmodel API (Netex), and the internal model now has a indirect dependency to the Transmodel API. Even if it looks good to have a method here and you avoid `instanceOf` it is not allowed. There are two solutions, eiter: 
- Create an enumeration and return that enum here and a mapper in the transmodel API
- Add a mapper in the Transmodel API witch uses instanceof. 

I would have done the latest, even if it is not "safe" when the domain model change. 

@optionsome Would the GTFS API benefit from having an enum here?

Reply:
The GTFS API is totally agnostic to the specific stop implementation and doesn't need the type (yet).

I would go with the mapper in Transmodel which uses instanceOf.